### ISSUE TITLE
x96 max plus no wifi show up

### ISSUE THREAD

**Issue by karnadii** [haiibo/OpenWrt#30]
no wifi for s905x3 x96 max plus for kernel 5.15 and 5.19

**Comment by haiibo** [comment]
Do you have a wireless option in your network

**Comment by karnadii** [comment]
> 

yes I have wireless option in network, but can not enable it.

**Comment by haiibo** [comment]
You try to change the channel to see if it can be used normally

**Comment by haiibo** [comment]
If it still cannot be used, it may be caused by the configuration change between the new production and the previous production. It is not recommended to use the built-in wifi.

**Comment by karnadii** [comment]
![Screenshot 2022-10-05 001807](https://user-images.githubusercontent.com/18657277/193884831-e14d1c4c-1616-46ed-bec7-c9961faa3a3d.png)
![Screenshot 2022-10-05 001842](https://user-images.githubusercontent.com/18657277/193884849-1558a98c-a62c-498c-b9a3-003e8e6bf7f4.png)

can not change anything. I am not using the wifi as my main wifi. just for debugging purpose. If my modem has problem I can use wifi as WWAN temporarily like that.

**Comment by haiibo** [comment]
Let me ask flippy,I use flippy's config file

**Comment by haiibo** [comment]
you try https://github.com/ophub/amlogic-s9xxx-openwrt/releases/download/OpenWrt_Aml_lede_10.04.0530/openwrt_s905x3_k5.15.71_2022.10.04.img.gz

**Comment by haiibo** [comment]
See if wifi can be used

**Comment by karnadii** [comment]
I have use that, and the wifi is working

**Comment by haiibo** [comment]
That should be that the wireless driver is not added,Do you know your wireless network card model

**Comment by karnadii** [comment]
rtl8822cs

**Comment by haiibo** [comment]
Can you use wifi firmware before is lede source code?

**Comment by karnadii** [comment]
ok let me try again to reflash the openwrt_aml_lede to make sure

**Comment by karnadii** [comment]
no, sorry again. my x96 max plus suddenly won't boot from sdcard anymore. can't test again until I fix this

**Comment by haiibo** [comment]
You need to compile the official firmware, I use lede, lede does not have 8822 driver

**Comment by karnadii** [comment]
```
0-03
───────────────────────────────────────────────────────────────────────
 Device Model: X96 MAX PLUS Q2, X96 Air Q1000
 Architecture: AArch64 : Cortex-A55 x 4
 Load Average: 0.04 0.19 0.11   Uptime: 5m 11s
 Ambient Temp: 61.8  °C         CPU Freq: 1800 Mhz
 Memory Usage: 5% of 3337M      Swap Usage: 0% of 0M   
 Boot Storage: 39% of 254.7M    ROOTFS: 87% of 960.0M
───────────────────────────────────────────────────────────────────────
root@OpenWrt:~# lsusb
Bus 001 Device 008: ID 05c6:6000 Qualcomm, Incorporated Qualcomm CDMA Technologies MSM
Bus 002 Device 001: ID 1d6b:0003 Linux 5.15.71-flippy-77+o xhci-hcd xHCI Host Controller
Bus 004 Device 001: ID 1d6b:0003 Linux 5.15.71-flippy-77+o vhci_hcd USB/IP Virtual Host Controller
Bus 001 Device 001: ID 1d6b:0002 Linux 5.15.71-flippy-77+o xhci-hcd xHCI Host Controller
Bus 003 Device 001: ID 1d6b:0002 Linux 5.15.71-flippy-77+o vhci_hcd USB/IP Virtual Host Controller
root@OpenWrt:~# cat /etc/openwrt_release
DISTRIB_ID='OpenWrt'
DISTRIB_RELEASE='SNAPSHOT'
DISTRIB_TARGET='armvirt/64'
DISTRIB_ARCH='aarch64_cortex-a53'
DISTRIB_TAINTS='no-all'
DISTRIB_SOURCECODE='lede'
DISTRIB_REVISION='R2022.09.29'
DISTRIB_DESCRIPTION='OpenWrt '
root@OpenWrt:~# 
```
this one has a working wifi. this is lede right?

**Comment by haiibo** [comment]
You try the newly compiled firmware, I added some wireless drivers

**Comment by karnadii** [comment]
> You try the newly compiled firmware, I added some wireless drivers

still not working. it is not crucial, so since this is not supported by lede I will close it, thanks for your time.
